FIX massaction builddoc: include PDF files created from an ODT template

pa-de-solminihac opened the pull request:
# Fix #9221
Include PDF files created from an ODT template when building a merged PDF file

Commit:
FIX massaction builddoc: include PDF files created from an ODT template too

eldy commented on htdocs/core/actions_massactions.inc.php:
```diff
@@ -825,7 +825,7 @@
 
 	$arrayofinclusion=array();
-	foreach($listofobjectref as $tmppdf) $arrayofinclusion[]='^'.preg_quote(dol_sanitizeFileName($tmppdf).'.pdf','/').'$';
+	foreach($listofobjectref as $tmppdf) $arrayofinclusion[]='^'.preg_quote(dol_sanitizeFileName($tmppdf),'/').'[a-zA-Z0-9-_]*\.pdf$';
```
If we do so, we will also include PDF that were added manually from the "Join files" tab because files have name
REF-xxx.pdf
When generated with ODX, filename is 
REF_nameofodx.pdf
So i try another fix

eldy committed:
Update actions_massactions.inc.php

Try another change to include PDF generated from ODX without including files uploaded manually.

pa-de-solminihac commented:
Thanks for fixing the fix ;)
